packages/lesswrong/components/jargon/JargonTooltip.tsx

```diff
@@ -158,9 +188,10 @@ export const JargonTooltip = ({term, definitionHTML, approved, deleted, humansAn
       forceOpen={open}
       analyticsProps={{ pageElementContext: 'jargonTermHovered' }}
       otherEventProps={{ term }}
+      
     >
       <LWClickAwayListener onClickAway={() => setOpen(false)}>
-        <span className={classes.jargonWord} onClick={clickTooltip}>
+        <span className={classNames(classes.jargonWord, postGlossariesPinned && classes.pinnedJargonWord)} onClick={clickTooltip}>
```

Comment:
Does this need to respect `postGlossariesPinned`, or the more complicated `displayAsPinned` (which we don't have here)?